components/style/values/specified/length.rs

```diff
@@ -1226,3 +1227,97 @@ impl LengthOrNumber {
         }
     }
 }
+
+/// A value suitable for a `min-width` or `min-height` property.
+/// Unlike `max-width` or `max-height` properties, a MinLength can be
+/// `auto`, and cannot be `none`.
+#[derive(Debug, Clone, PartialEq)]
+#[cfg_attr(feature = "servo", derive(HeapSizeOf))]
+#[allow(missing_docs)]
+pub enum MinLength {
```

Comment:
The comment just above this snippet explains that `MinLength` can be `auto` but not `none`, and `MaxLength` can be `none` but not `auto`.